### ISSUE TITLE
Classification with Inception fails with BLAS SGEMM Error

### ISSUE THREAD

**Issue by Robmosh91** [keras-team/keras#8376]
Hi,

i am trying to train the Inception V3 net with the pretrained weights for multilabel classification.
For this I have a numpy array with 1000 train images where each is preprocessed and resized and a numpy array with shape (1000,36) where each image can have mutliple labels.

When training a standard image classification everything works just fine an no error accures, but with the multi label data I always get the error

InternalError (see above for traceback): Blas SGEMM launch failed : m=63948, n=80, k=64
	 [[Node: conv2d_4/convolution = Conv2D[T=DT_FLOAT, data_format="NHWC", padding="VALID", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/gpu:0"](max_pooling2d_1/MaxPool, conv2d_4/kernel/read)]]
	 [[Node: metrics/acc/Mean_1/_2527 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/cpu:0", send_device="/job:localhost/replica:0/task:0/gpu:0", send_device_incarnation=1, tensor_name="edge_8037_metrics/acc/Mean_1", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/cpu:0"]()]]

I get this error when using transfer learning with the pretrained image net weights as well as when finetuning.
Anybody has done this before and has an idea how to get around this problem and train multilabel classification with keras and the inception V3 model?

Thanks,

Robert

Update: Now also the example with the single label classification fails with the same error.
